auto2cmake.py

```diff
@@ -1051,7 +1051,7 @@ def process_configure_ac(fname):
 ########################################################################################################################
 def generate_default_cmake(req_dir):
     projname = req_dir.split("/")[-1] + ")\n"
-    sources = "set (project "
+    sources = "project("
```

Comment:
This won't work, because **project** is a builtin keyword (https://cmake.org/cmake/help/latest/command/project.html) while the code which was there before created a new variable called `project`, which was used below, in `set(${project}_SOURCES .....`, Now, we do not create that variable, so code below will not work as expected.

Reply:
I will revert it for now.

Reply:
:+1: